tests/scripts/exec_template.yml

```diff
@@ -0,0 +1,14 @@
+# ansible-playbook ../tests/drop_counters/exec_template.yml -i lab -l DUT_HOSTNAME --extra-vars "VAR_EXAMPLE=VALUE VAR_EXAMPLE=VALUE" -vvvvv
```

Comment:
I have added support for Onyx OS based fanout. But also left this approach to execute any of ansible playbook on fanout because it is need when existed ansible modules like "onyx_config or onyx_command" does not support some  functionality. Currently it does not support extended/additonal docker commands over Onyx CLI, and it was required to left such workaround.